submissions/madmaxWMFU/google-popup/index.html

```diff
@@ -0,0 +1,186 @@
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -2070px"></span>
+                                    <span class="textApp">Blogger</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1587px"></span>
+                                    <span class="textApp">Hangouts</span>
+                                </a>
+                            </div>
+
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -897px"></span>
+                                    <span class="textApp">Jamboard</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -621px"></span>
+                                    <span class="textApp">Class</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1863px"></span>
+                                    <span class="textApp">Earth</span>
+                                </a>
+                            </div>
+                            <div class="app">
+                                <a href="#">
+                                    <span class="iconApp" style="background-position:0 -1725px"></span>
+                                    <span class="textApp">Collections</span>
+                                </a>
+                            </div>
+                        </div>
+                        <div class="linkToOtherApps">
+                            <a href="#">Other products from G00gle</a>
+                        </div>
+                        <div class="otherApps">
+                            <label for="moreApps">More</label>
+                        </div>
+                    </div>
+                    <div class="cornerTop"></div>
+                    <div class="cornerButtom"></div>
+                </div>
+                <div>
+                    <a id="ring" class="link ring" href="#" title="Notification: 0"></a>
+                </div>
+                <div>
+                    <a id="account" class="link account" href="#"></a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container">
```

Comment:
Try to use more semantic tags. 
Hint: in this case better use `<main>`

Reply:
Done!